deploy/non-olm/v1.2.2/controller-4.yml

```diff
@@ -15,7 +15,5 @@ spec:
   mig_pod_limit: 100
   mig_namespace_limit: 10
 
-  mig_failure_rollback: false
```

Comment:
Was this meant for latest?

Reply:
Yeah it looks like we also got this change into latest, but it should _only_ go into latest, not into v1.2.2 @aufi

Reply:
Good catch, I did not notice that both were in controller-4.yml. We want it in latest/controller-3.yml and latest/controller-4.yml? Technically the controller can run on either. I can see where having it in both would be confusing and is why I generally leave things in defaults for people to override, but I leave it up to you guys.

Reply:
Thanks for the catch! Updated.